components/Token/ContentRenderer.tsx

```
@@ -39,18 +40,24 @@ const ContentRenderer = ({ metadata }: ContentRendererProps) => {
         />
       </div>
     );
+  if (mimeType.includes("text/plain"))
+    return (
+      <div
+        className="font-archivo bg-grey-moss-100 p-2 rounded-md text-md !normal-case"
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)